tests/extmod/vfs_posix_nonwindows.py

```diff
@@ -0,0 +1,32 @@
+# Test for VfsPosix error conditions that cannot be reproduced on Windows
```

Comment:
I think this test should be called `vfs_posix_enoent.py`.  It may eventually run on windows, or may not run on other systems eg macOS.

Reply:
Done. (It does currently run on macOS, that’s where I’m developing.)